NickvisionTagger.Shared/Models/MusicFile.cs

```diff
@@ -360,11 +360,9 @@ public string Fingerprint
                     }
                 };
                 _fpcalc.Start();
+                _fpcalc.WaitForExit(TimeSpan.FromSeconds(8));
                 _fingerprint = _fpcalc.StandardOutput.ReadToEnd();
-                if (!_fpcalc.WaitForExit(TimeSpan.FromSeconds(10)))
-                {
-                    _fpcalc.Kill();
-                }
+                _fpcalc.Kill();
```

Comment:
If the process has already exited, this will cause [InvalidOperationException](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill?view=net-7.0#system-diagnostics-process-kill), you need try-catch [like in Cavalier](https://github.com/NickvisionApps/Cavalier/blob/main/NickvisionCavalier.Shared/Models/CAVA.cs#L94)

Reply:
it works fine for me still...but I will add it in anyway just in case